README.md

```diff
@@ -1,133 +1,57 @@
-# FLAME GPU 2 Submodel Benchmark Model
+# FLAMEGPU2 Submodel Benchmark
 
-This repository contains performance benchmarking of a [FLAMEGPU/FLAMEGPU2](https://github.com/FLAMEGPU/FLAMEGPU2) implementation of the SugarScape model, used to benchmark the [Submodel](https://docs.flamegpu.com/guide/7-submodels/) feature of FLAME GPU 2.
+This repository contains performance benchmarking of a [FLAME GPU 2](https://github.com/FLAMEGPU/FLAMEGPU2) implementation of a sugarscape model with instantaneous grow back. It is an exmaple which shows how to use sub modelling to resolve compentation within the model. In this case competition for movement to areas of high resource.
```

Comment:
`compentation` -> `competition`

Reply:
fixed with a couple of other typos spotted